core/spec.md

```diff
@@ -1263,6 +1379,72 @@ HTTP body.
 
 ---
 
+### `xid` Resolution API
+
+The resolution API is used to translate an `xid` into a URL that can be used
+to access an xRegistry entity. An implementation of the registry API SHOULD
+provide a resolution API under the path `/resolution`.
+The resolution API MAY also be offered independently of the registry API.
+It can resolve `xids` and federate access to multiple registries.
+In the following, `/` is the root of the resolution API, even if it would be
+provided under `/resolution` as part of an registry.
+
+#### `GET /xid/{xid}` and `HEAD /xid/{xid}`
+
+The placeholder `{xid}` depicts the `xid` to resolve. As `xids` MAY be full
+URIs or URI references, the `xid` MUST be percent-encoded accordingly.
+
+For `GET` and `HEAD` requests, the server MUST respond as follows:
+
+- If the `xid` is not found, the server MUST respond with an HTTP
+  `404 Not Found`.
+- If the `xid` is found and there exists exactly one URL to access the entity,
+  the server MUST respond with an HTTP `308` where the `Location` header
+  MUST contain the URL to access the entity.
+- If the `xid` is found and there are multiple URLs to access the entity, the
+  server MUST respond with an HTTP `300 Multiple Choices` where the choices
+  are listed as `Link` headers. Each `Link` header MUST contain a
+  `rel="alternate"`as specified
+  in [RFC5988](https://tools.ietf.org/html/rfc5988),
+[RFC8288](https://tools.ietf.org/html/rfc8288)
+  and
+  [RFC7231](https://tools.ietf.org/rfc/rfc7231#section-6.4.1). If the server
+  has a preferred URL, e.g. the authoritative URL, it SHOULD be returned as
+  `Location` header.
+
+If the applied request method is `GET`, the body of the response MUST be a
+JSON object as follows:
+
+```yaml
+{
+  "xid": "URI",     # The xid to resolve
+  "locations": [
+    {
+      "url": "URL",   # The URL to access the entity
+      "authoritative": boolean
+    } *
+  ],
+  "externalids":[
+    {
+      "type": "STRING",  # The type of the external ID
+      "id": "STRING"         # External IDs that are equivalent to the xid"
+    } *
+  ] ?
+}
+```
+
+#### `locations`
+
+The `locations` array contains the URLs to access the entity. In addition to
+the `url`, the `authoritative` attribute indicates whether the URL points to
+the registry that is authoritative for the entity.
```

Comment:
I think you need to define "authoritative" w/o using the word "authoritative". Do you really mean that the response can be trusted (https://www.merriam-webster.com/dictionary/authoritative)? Or do you mean that this is the "original source of truth" for this entity (meaning, all others are just aliases/pointers) ?